contracts/BridgeEscrow.sol

```diff
@@ -1,258 +1,280 @@
 //SPDX-License-Identifier: Unlicense
-pragma solidity ^0.8.0;
+
+pragma solidity 0.8.4;
 
 import "hardhat/console.sol";
+import "@openzeppelin/contracts/security/ReentrancyGuard.sol";
 import "./OLToken.sol";
 
-contract BridgeEscrow {
-
+contract BridgeEscrow is ReentrancyGuard {
     struct AccountInfo {
         // user address on this chain
         // eth->0L transfer
-        address sender_this;
+        address senderThis;
         // user address on the other chain
         // eth->0L transfer
-        bytes16 sender_other;
+        bytes32 senderOther;
         // user address on the other chain
         // receiver address on eth chain
         // 0L->eth transfer
-        address payable receiver_this;
+        address payable receiverThis;
         // receiver address on eth chain
         // eth->0L transfer
-        bytes16 receiver_other;
+        bytes32 receiverOther;
         // value sent
-        uint64 balance;
+        uint256 balance;
```

Comment:
`unit256` and `bytes32` is what the EVM gives you as default. Lowering the types makes it actual more gas costly in this instance.